### PR TITLE
Clean up LISK32 charset handling

### LINKED ISSUE #7927: Clean up LISK32 charset handling
There are a few things in Cryptography element which managed to escape from a code review:

* `validateLisk32Address()` returns true or never instead of only throwing like most other validate functions
* There are 2 constants in `address.ts` defining what BASE32 charset is: `CHARSET` and `LISK32_CHARSET`
* Constants from `address.ts` could be moved to `constants.ts`
* Error messages are not using values from constants but hardcoded literals
* Test assertions do not need `return` statement

### LINKED ISSUE #8653: Duplicate constants for Lisk32 charset
### Description

The [`CHARSET`](https://github.com/LiskHQ/lisk-sdk/blob/89e7504ef5eb6183aefe576a93be3d6052e56038/elements/lisk-cryptography/src/address.ts#L19) and [`LISK32_CHARSET`](https://github.com/LiskHQ/lisk-sdk/blob/89e7504ef5eb6183aefe576a93be3d6052e56038/elements/lisk-cryptography/src/address.ts#L121) constants are equivalent and are both used in the same set of functions for validation and encoding/decoding Lisk32 addresses. The code should be simplified to define and use only one constant for the valid set of characters.

### Which version(s) does this affect? (Environment, OS, etc...)

`v6.0.0-beta.2`

### PR DESCRIPTION
**Pull request by bobanm**
### What was the problem?

This PR resolves #7927 and resolves #8653

### How was it solved?

* Replaced duplicated `CHARSET` and `LISK32_CHARSET` constants by `BASE32_CHARSET` constant and moved to `constants.ts`
* Error messages are now using values from constants instead of hardcoded literals
* BONUS: Removed duplicated `lisk-cryptography` test suite from `lisk-client` 😎 
* BONUS: Removed return statements from test assertions 😎

Following [suggestion](https://github.com/LiskHQ/lisk-sdk/pull/7928#discussion_r1059877664) from @shuse2, I left `validateLisk32Address()` to throw when validation fails and return `true` when validation succeeds, in order not to introduce a breaking change for SDK users.

### How was it tested?

All tests OK 👌🏻

### CODE REVIEW

File: elements/lisk-cryptography/src/constants.ts [hunk 1]
```diff
@@ -15,11 +15,12 @@
...
 export const DEFAULT_LISK32_ADDRESS_PREFIX = 'lsk';
-export const DEFAULT_BASE32_ADDRESS_PREFIX = DEFAULT_LISK32_ADDRESS_PREFIX;
 export const ED25519_CURVE = 'ed25519 seed';
 export const MAX_UINT32 = 4294967295;
 export const HARDENED_OFFSET = 0x80000000;
 export const HASH_LENGTH = 32;
 export const L = 8160; // 32 * 255
 export const EMPTY_SALT = Buffer.from(new Array(32).fill(0));
 export const SHA256 = 'sha256';
+export const BASE32_CHARSET = 'zxvcpmbn3465o978uyrtkqew2adsjhfg';
```
Comment (shuse2) [inline elements/lisk-cryptography/src/constants.ts:25]: We don't need to expose this charset

File: elements/lisk-cryptography/src/address.ts [hunk 2]
```diff
@@ -138,19 +137,17 @@ export const validateLisk32Address = (
...
 	}
 
-	const integerSequence = addressSubstringArray.map(char => LISK32_CHARSET.indexOf(char));
+	const integerSequence = addressSubstringArray.map(char => BASE32_CHARSET.indexOf(char));
 
 	if (!verifyChecksum(integerSequence)) {
 		throw new Error('Invalid checksum for address.');
 	}
-
-	return true;
```
Comment (shuse2) [inline elements/lisk-cryptography/src/address.ts:153]: this is breaking change, and it might cause the other tools to fail

**Review by shuse2 — CHANGES_REQUESTED** [review]

File: elements/lisk-cryptography/src/address.ts (hunk 2 above)
Comment (bobanm) [reply elements/lisk-cryptography/src/address.ts]: I personally find it quite inconsistent that a function throws when validation fails, returns true when everything validation succeeds, and never returns false. But I have reverted to the previous state.

File: elements/lisk-cryptography/src/constants.ts (hunk 1 above)
Comment (bobanm) [reply elements/lisk-cryptography/src/constants.ts:25]: The reason I moved it to `constants.ts` and exported it is to be able to use it in tests.

**Review by shuse2 — APPROVED** [review]

**Review by mosmartin — APPROVED** [review]

**Pull request merged**